samples/drivers/led/pwm/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+config BLINK_DURATION_SHORT
+	int "LED on and off times for short-cycle blinking demo"
+	default 100
+	help
+	  Specifies the LED blinking time in milliseconds in the short-cycle
```

Comment:
And maybe: Specifies the LED on/off delay in milliseconds for short cycle blinking demonstration.

Reply:
I applied it.